slack/web/base_client.py

```diff
@@ -56,7 +56,9 @@ def _get_event_loop(self):
             asyncio.set_event_loop(loop)
             return loop
 
-    def _get_headers(self, has_json, has_files, request_specific_headers):
+    def _get_headers(
+        self, has_json: bool, has_files: bool, request_specific_headers: Optional[dict]
```

Comment:
If the type is an optional dict then I believe we need to set the default value here to "None" as well.